src/background.js

```diff
@@ -849,7 +849,14 @@ function onMessageHandler(request, sender, callback) {
 		return false;
 	}
 	if (name === 'account.openCheckoutPage') {
-		utils.openNewTab({ url: `https://checkout.${globals.GHOSTERY_DOMAIN}.com/plus`, become_active: true });
+		let url = `https://checkout.${globals.GHOSTERY_DOMAIN}.com/plus`;
+		const { from } = message;
```

Comment:
Rather than checking for the source, let's just check to see if any utms are passed along with the message and append to the url string

Reply:
I like that this removes the if statement on a magic string